src/architecture/configuration.md

```diff
@@ -42,14 +42,25 @@ This categorization holds both for core and service parameters.
     | Global  | Validators’ public keys | Anchoring address |
     | Local   | Validator’s private key | RPC params for Bitcoin Core |
 
+!!! tip
+    See [sample configuration file](https://github.com/exonum/exonum/blob/master/exonum/tests/testdata/config/config02.toml)
```

Comment:
I'm against linking to an external file in this form, because it makes the documentation fragile - the file may change, so the documentation becomes *weird*. I'd advise to copy the file contents to the article, as it was previously. Alternatively, you may specify a commit or tag `v0.1` for the file.